src/material/button-toggle/button-toggle.ts

```diff
@@ -83,7 +83,8 @@ export const MAT_BUTTON_TOGGLE_GROUP_VALUE_ACCESSOR: any = {
   multi: true
 };
 
-let _uniqueIdCounter = 0;
+// Counter used to generate unique IDs.
+let uniqueIdCounter = 0;
```

Comment:
Just for clarification, could you fill me in on why we are making this public? Or is there another reason we're removing the underscore?

Reply:
I didn't make it public, I just removed the underscore since the constant isn't being exported so the name doesn't really matter.